src/index.ts

```diff
@@ -81,9 +87,18 @@ export async function extract(
 
   const parser = new InstructionParser(programId);
   const events = getEvents(program, tx);
```

Comment:
Looks unused variable now, and `getEvents` function too

Reply:
We are still testing the new parser results with the live transactions. After fixing any issues found while testing, I would update this to clean up events parsing code altogether.